rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -775,7 +775,7 @@ void main_window::HandlePackageInstallation(QStringList file_paths)
 		pdlg.show();
 
 		Emu.SetForceBoot(true);
-		Emu.Stop();
+		Emu.Kill();
```

Comment:
while this is correct, it is missing a call to:

	if (!m_gui_settings->GetBootConfirmation(this))
	{
		return;
	}
        followed by Graceful exit.
in InstallPackages imo

Reply:
this is still missing